test/e2e/logging_test.go

```diff
@@ -0,0 +1,102 @@
+package e2e
+
+import (
+	"testing"
+
+	operatorsv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/console-operator/pkg/testframework"
+)
+
+// TestDebugLogLevel sets 'Debug' LogLevel on the console operator and tests
+// if '--log-level=*=DEBUG' flag is set on the console deployment
+func TestDebugLogLevel(t *testing.T) {
+	client := testframework.MustNewClientset(t, nil)
+	defer testframework.SetLogLevel(t, client, operatorsv1.Normal)
+	testframework.MustManageConsole(t, client)
+	testframework.MustNormalLogLevel(t, client)
+
+	err := testframework.SetLogLevel(t, client, operatorsv1.Debug)
+	if err != nil {
+		t.Fatalf("error: %s", err)
+	}
+
+	deployment, err := testframework.GetConsoleDeployment(client)
+	if err != nil {
+		t.Fatalf("error: %s", err)
+	}
+	flagToTest := getFlag(operatorsv1.Debug)
+	if !isFlagInCommand(t, deployment.Spec.Template.Spec.Containers[0].Command, flagToTest) {
+		t.Fatalf("error: flag not found in command %v \n", deployment.Spec.Template.Spec.Containers[0].Command)
+	}
+}
+
+// TestTraceLogLevel sets 'Trace' LogLevel on the console operator and tests
+// if '--log-level=*=TRACE' flag is set on the console deployment
+func TestTraceLogLevel(t *testing.T) {
+	client := testframework.MustNewClientset(t, nil)
+	defer testframework.SetLogLevel(t, client, operatorsv1.Normal)
+	testframework.MustManageConsole(t, client)
+	testframework.MustNormalLogLevel(t, client)
+
+	err := testframework.SetLogLevel(t, client, operatorsv1.Trace)
+	if err != nil {
+		t.Fatalf("error: %s", err)
+	}
+
+	deployment, err := testframework.GetConsoleDeployment(client)
+	if err != nil {
+		t.Fatalf("error: %s", err)
+	}
+	flagToTest := getFlag(operatorsv1.Trace)
+	if !isFlagInCommand(t, deployment.Spec.Template.Spec.Containers[0].Command, flagToTest) {
+		t.Fatalf("error: flag not found in command %v \n", deployment.Spec.Template.Spec.Containers[0].Command)
+	}
+}
+
+// TestTraceLogLevel sets 'TraceAll' LogLevel on the console operator and tests
+// if '--log-level=*=TRACE' flag is set on the console deployment
+func TestTraceAllLogLevel(t *testing.T) {
+	client := testframework.MustNewClientset(t, nil)
+	defer testframework.SetLogLevel(t, client, operatorsv1.Normal)
+	testframework.MustManageConsole(t, client)
```

Comment:
Do we need the `MustManageConsole`?
Thinking about this.  Ideally all of our tests would be black boxed.  Or perhaps share a `Setup()` func in this suite?